quarkchain/tools/miner_address.py

```diff
@@ -7,7 +7,7 @@
 from utils import underline, colorify
 from quarkchain.utils import sha3_256
 
-FILE = "../../testnet/2.5/cluster_config_template.json"
+FILE = "../mainnet/singularity/cluster_config_template.json"
```

Comment:
wait, I think the path should be relative to current script, so should be `../../mainnet/xxx`?

Reply:
You are right! there are one more dir before pyquarkchain